Microsoft.DotNet.Interactive.PowerShell/PowerShellExtensions.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Interactive.PowerShell
+{
+    using System.Management.Automation;
```

Comment:
why nested namespace?

Reply:
The nested namespace is because PowerShell is both a part of a namespace, and a type.